internal/engine/buildcontroller_test.go

```diff
@@ -732,7 +732,7 @@ func TestBuildControllerManualTriggerWithFileChangesSinceLastSuccessfulBuildButB
 
 	f.podEvent(basePB.Build())
 
-	f.b.nextBuildError = errors.New("build failure!")
+	f.SetNextBuildError(errors.New("build failure!"))
```

Comment:
while debugging an issue, I saw this comment, and figured I might as well act on it:
```
	// Set this to simulate a build with no results and an error.
	// Do not set this directly, use fixture.SetNextBuildError
	nextBuildError error
```